packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -31,13 +32,21 @@ export type CallControlsProps = {
 /**
  * @private
  */
-export const CallControls = (props: CallControlsProps): JSX.Element => {
+export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX.Element => {
   const options = useMemo(() => (typeof props.options === 'boolean' ? {} : props.options), [props.options]);
   const customButtons = useMemo(
     () => generateCustomButtons(onFetchCustomButtonPropsTrampoline(options), options?.displayType),
     [options]
   );
 
+  // This useState and useEffect is to rerender the control bar when the container height and/or width change
```

Comment:
Is this necessary? My understanding is if the props passed in are changed the component should re-render (perhaps react is smart and avoids the re-render if the props are unused inside the component..)

Reply:
Yes. React is smart enough. Thanks